README_MEW.md

```diff
@@ -0,0 +1,235 @@
+### [https://www.MyEtherWallet.com](https://www.MyEtherWallet.com)
```

Comment:
My suggestion is to keep the README file. An additional file might just confuse people on where to find the documentation.